### PR TITLE
Day3 Assignment

### PR DESCRIPTION
**Pull request by Monalisa-prog**

### CODE REVIEW

**Comment by Monalisa-prog** [comment]
I have doubt on quesno7,9 and 10

File: Day3/10Ques.js
Comment (NaveenMohanty) [inline Day3/10Ques.js:1]: Pls don't push those file whose code you have not done

File: Day3/7Ques.js
Comment (NaveenMohanty) [inline Day3/7Ques.js:1]: Uncomment the code part

File: Day3/8Ques.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+//Question8: Write a program that takes an array of strings and returns a new array with all the strings converted to uppercase using a loop.
+
+arr = ["apple", "banana", "cherry"];
```
Comment (NaveenMohanty) [inline Day3/8Ques.js:3]: It should be let arr. Pls run the code before you push

File: Day3/9Ques.js
Comment (NaveenMohanty) [inline Day3/9Ques.js:1]: Delete this file

**Review by NaveenMohanty — COMMENTED** [review]
Pls all the mentioned issue and msg in whatsapp

**Review by NaveenMohanty — COMMENTED** [review]
Pls push resolved issues and ping in the whatsapp

**Pull request merged**